### PR TITLE
Small Library app to test Javascript OOP features

### PR DESCRIPTION
**Pull request by MkrtichSargsyan**
In this project, we build a small Library app to show and add books.

1. Bookmark your books as favorite and set if you have already read it or not. There are options to filter the books in the ones you compleated or the ones that not.​

2. Every move is saved in the local storage of your browser.

### CODE REVIEW

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team. 💻 👀 ✨ 
- Good work structuring your code properly.

## However, there are some changes required 👀 🧰 🔨 🔧 
- When I try to delete a single book, it does initially disappear from the page. But if I click on any of the tabs (All books, Not finished yet books, Finished books), the book appears again. Please fix this.
- Your Live Demo link is broken. Please fix it.

**[ OPTIONAL ]**
- I'm able to add books with blank titles. Please validate all input fields before adding new books.
- You may add a data persistence method such as Firebase or localStorage.
### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Comment by emasdev** [comment]
- Issue with removed books is already fixed

- "You may add a data persistence method such as Firebase or localStorage." 
We are using local storage to persist the data in the library.js file

File: scripts/helper.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      renderFavoriteInfo(arr);
+    });
+
+    removeBtn.addEventListener('click', () => {
+      arr = arr.filter((el) => el !== book);
+      document.getElementById('bookTable').innerHTML = '';
+      renderBooks(arr);
+      renderBooksInfo(arr);
+      localStorage.setItem('myLibrary', JSON.stringify(arr));
+    });
```
Comment (MuriloRoque) [inline scripts/helper.js:96]: - [ ] The bug still persists when I click this button. When I try to remove a book using it, it disappears, but when I refresh the page or click somewhere else, it's still there. Also, I checked the `localStorage` using the console and it doesn't change when I click this button. Please try debugging this and fix this issue.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @MkrtichSargsyan and @emasdev :wave: 

### Good job fixing almost all of the issues pointed by the last TSE. There's still one bug we need to fix, please check my comment on your code and the request below:

- [ ] Your Pull Request is currently not checking for linter errors, please fix this issue.

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!


- [ ] `OPTIONAL` I can still fill in blank inputs, the only one that wouldn't be accepted if blank is the "cover" field, please fix it for all of them, also, it would be better if you can add an error message so a user would know what's the issue.

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**